### PR TITLE
Updates release notes for v11.1 for  function removal.

### LINKED ISSUE #2624: Consider adding instructions for removing import of `convert_point_data`
## Describe the Task ##
In dtcenter/MET#2509, the location of the `convert_point_data` function has been moved. We should consider providing details in the upcoming release notes. I will note, that it's likely this affects a very, very small number of users but it's hard to know exactly.

Previously, this used to live in `share/met/python/met_point_obs.py`, and a user accessed this function in their Python embedding script like this:

```
from met_point_obs import convert_point_data
met_point_data = convert_point_data(point_data)
```

This no longer works. The user can now remove both of the above lines, and MET automatically reformats `point_data` into the correct format if it is not already. BUT, the user MUST have a variable named `point_data`. For instance, if their script looked like this:
```
from met_point_obs import convert_point_data
my_11col_data = pd.DataFrame('foo')
met_point_data = convert_point_data(my_11col_data)
```

They must also change this line:
`my_11col_data = pd.DataFrame('foo')`
to be:
`point_data = pd.DataFrame('foo')`

@DanielAdriaansen will add a few sentences to "upgrade instructions" for this major release describing this. Make these changes in the main_v11.1 branch (here: https://met.readthedocs.io/en/main_v11.1/Users_Guide/release-notes.html#met-version-11-1-0-upgrade-instructions). This section is transient and won't apply to future versions so it will be nice to put it here. Use feature branch because this branch is protected so make a PR.

### Time Estimate ###
30 minutes

### Assignee ###
- [ ] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [ ] Select **component(s)**
- [ ] Select **priority**
- [ ] Select **requestor(s)**

### Projects and Milestone ###
- [ ] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED CYCLE ASSIGNMENT** label
- [ ] Select **Milestone** as the next official version or **Future Versions**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [ ] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Task Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issues
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by DanielAdriaansen**
Closes #2624.

## Expected Differences ##

- [ ] Do these changes introduce new tools, command line arguments, or configuration file options? **[Yes or No]**</br>
If **yes**, please describe:</br>

- [ ] Do these changes modify the structure of existing or add new output data types (e.g. statistic line types or NetCDF variables)? **[Yes or No]**</br>
If **yes**, please describe:</br>

## Pull Request Testing ##

- [ ] Describe testing already performed for these changes:</br>

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ ] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes or No]**

- [ ] Do these changes include sufficient testing updates? **[Yes or No]**

- [ ] Will this PR result in changes to the test suite? **[Yes or No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR definition above.
- [ ] Ensure the PR title matches the feature or bugfix branch name.
- [ ] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

**Comment by DanielAdriaansen** [comment]
I am trying to view the docs here: https://met.readthedocs.io/en/hotfix_update_v11.1_release_notes, but it doesn't seem to be working.

**Comment by jprestop** [comment]
@DanielAdriaansen Only branches that begin with the following prefixes have documentation generated automatically:
      - develop
      - develop-ref
      - feature_*
      - main_*
      - bugfix_*

as listed in the GitHub Actions workflow file https://github.com/dtcenter/MET/blob/main_v11.0/.github/workflows/documentation.yml

Since this branch begins with "hotfix" the documentation was not automatically generated.  However, the PR generates a build of documentation, which you can see here:
https://met--2630.org.readthedocs.build/en/2630/Users_Guide/release-notes.html

File: docs/Users_Guide/release-notes.rst [hunk 1]
```diff
@@ -278,6 +278,14 @@ MET Version 11.1.0 upgrade instructions
 
 * If running TC-Pairs to generate consensus tracks, update your TC-Pairs configuration file to include the new **write_members** option (`#2429 <https://github.com/dtcenter/MET/issues/2429>`_).
 
+* If using Python embedding for point observations, and the following lines of code are in your Python script please remove them and refactor your Python script following the instructions in the User's Guide for Python embedding for point observations (`https://met.readthedocs.io/en/main_v11.1/Users_Guide/appendixF.html#python-script-requirements-for-point-observations`):
+
+.. code-block:: Python
```
Comment (jprestop) [inline docs/Users_Guide/release-notes.rst]:
```suggestion
  .. code-block:: Python
```

File: docs/Users_Guide/release-notes.rst [hunk 2]
```diff
@@ -278,6 +278,14 @@ MET Version 11.1.0 upgrade instructions
...
+ 
+  from met_point_obs import convert_point_data
```
Comment (jprestop) [inline docs/Users_Guide/release-notes.rst]:
```suggestion
    from met_point_obs import convert_point_data
```

File: docs/Users_Guide/release-notes.rst [hunk 3]
```diff
@@ -278,6 +278,14 @@ MET Version 11.1.0 upgrade instructions
...
+  ...
```
Comment (jprestop) [inline docs/Users_Guide/release-notes.rst]:
```suggestion
    ...
```

File: docs/Users_Guide/release-notes.rst [hunk 4]
```diff
@@ -278,6 +278,14 @@ MET Version 11.1.0 upgrade instructions
...
+  met_point_data = convert_point_data(point_data)
```
Comment (jprestop) [inline docs/Users_Guide/release-notes.rst]:
```suggestion
    met_point_data = convert_point_data(point_data)
```

**Comment by jprestop** [comment]
@DanielAdriaansen I added some suggestions for indention to align the code block with the bulleted list.

File: docs/Users_Guide/release-notes.rst [hunk 5]
```diff
@@ -278,6 +278,16 @@ MET Version 11.1.0 upgrade instructions
...
 
+* If using Python embedding for point observations, and the following lines of code is in your Python script:
+
+.. code:: Python
+ 
+   from met_point_obs import convert_point_data
+   ...
+   met_point_data = convert_point_data(point_data)
+
+Please remove those lines, and refactor your Python script following the instructions in the User's Guide for Python embedding for point data: `https://met.readthedocs.io/en/main_v11.1/Users_Guide/appendixF.html#python-script-requirements-for-point-observations`.
```
Comment (jprestop) [inline docs/Users_Guide/release-notes.rst]:
```suggestion
  Please remove those lines, and refactor your Python script following the instructions in the 
  User's Guide for Python embedding for point data: 
  `https://met.readthedocs.io/en/main_v11.1/Users_Guide/appendixF.html#python-script- 
  requirements-for-point-observations`.
```

**Comment by DanielAdriaansen** [comment]
> @DanielAdriaansen I added some suggestions for indention to align the code block with the bulleted list.

Thanks @jprestop! I am happy with things now. Feel free to approve based on how the docs look here:
https://met--2630.org.readthedocs.build/en/2630/Users_Guide/release-notes.html#met-version-11-1-0-upgrade-instructions

And then merge. Appreciate your help!

**Review by jprestop — APPROVED** [review]
Great @DanielAdriaansen!  It looks good to me too.  Thanks for your work on this task!  I approve this request.

**Pull request merged**